script_templates/ink_template.gd

```diff
@@ -0,0 +1,106 @@
+# ############################################################################ #
```

Comment:
If I understand the template system correctly, we might wanna add this file to the `.gitignore` or symlink it somehow?

I'm assuming it will be copied over everytime the plugin is activated, so ignoring it might be enough.

Reply:
Like I said, this is my first Pull Request, so I'm going to need some help understanding some things. So the `.gitignore` will tell Github to ignore the generated template, will this stop it from being added to the project page? Or will it stop it from being updated? 
It is generated as a copy of the template, so I believe... Actually, I don't think it would get overridden if the project was downloaded and opened. If the InkGD plugin was deactivated then reactivated it would be replaced, but not when the project opens for the first time. I'll have to look into that.

Reply:
> Like I said, this is my first Pull Request, so I'm going to need some help understanding some things.

All good.

> will this stop it from being added to the project page? Or will it stop it from being updated?

An ignored file isn't tracked by Git at all, so it won't appear in this repository, but it won't prevent local updates.

> Actually, I don't think it would get overridden if the project was downloaded and opened. If the InkGD plugin was deactivated then reactivated it would be replaced, but not when the project opens for the first time.

Since the template is copied over when `_enter_tree()` is invoked, I would assume that happens every time the editor is launched or the addon is updated/activated. Let me know what you find!

Reply:
OK, so adding `script_templates` to the .gitignore file will prevent the folder from appearing in the repository. That seems like the neatest solution. I'll try to add it.